examples/Makefile

```diff
@@ -108,6 +109,10 @@ rdkafka_zookeeper_example: ../src/librdkafka.a rdkafka_zookeeper_example.c
 	@echo "# More usage options:"
 	@echo "./$@ -h"
 
+openssl_engine_example_cpp: ../src-cpp/librdkafka++.a ../src/librdkafka.a openssl_engine_example.cpp
+	$(CXX) $(CPPFLAGS) $(CXXFLAGS) openssl_engine_example.cpp -o $@ $(LDFLAGS) \
+		../src-cpp/librdkafka++.a ../src/librdkafka.a $(LIBS)
```

Comment:
Not strictly related to this part, but a general question is:
Can OpenSSL engines be used when OpenSSL is statically linked into librdkafka/application?

Reply:
yes application should be able to use engine with "statically linked openSSL application"